packages/rx_bloc_cli/README.md

```diff
@@ -27,6 +27,7 @@ Create a new project at the given directory with lots of already set-up features
 |  `--enable-feature-deeplinks` |                   `false`                   | Enables Deep Links showcase flow for the project |
 |  `--enable-feature-widget-toolkit` |                   `false`                   | Enables widget_toolkit package showcase feature for the project |
 |  `--enable-social-logins` |                   `false`                   | Integrate social login with Apple, Google and Facebook functionality into our application |
+|  `--enable-internationalisation` |                   `true`                    |                       Enables internationalisation for the project                        |
```

Comment:
Let's change the name of the flag, since the template supports the internatialisation re regardless of the value of this flag. How about --enable-change-language ?

Reply:
`--enable-internationalisation` was changed to `--enable-change-language`